srcpkgs/smpq/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'smpq'
+pkgname=smpq
+version=1.6
+revision=1
+build_style=cmake
+configure_args="-DWITH_KDE=OFF"
+hostmakedepends="StormLib"
+makedepends="bzip2-devel zlib-devel StormLib-devel"
+short_desc="StormLib MPQ archiving utility"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://launchpad.net/smpq"
+distfiles="https://launchpad.net/smpq/trunk/${version}/+download/smpq_${version}.orig.tar.gz"
+checksum=b5d2dc8a5de8629b71ee5d3612b6e84d88418b86c5cd39ba315e9eb0462f18cb
+
+do_build() {
```

Comment:
This does nothing, why is it here?

Reply:
The software is old, you have to go into the folder to build some files and then another folder to build the software, cmake doesn't do that.

Reply:
But this literally does not build it, this CDs into the directory and then exits.  It gets built in the do_install phase, which is unaffected by this.

Reply:
This method of entering the folder and leaving the folder, avoids this error message and completes the package creation process:

=> ERROR: smpq-1.6_1: PKGDESTDIR is empty and build_style != meta
=> ERROR: smpq-1.6_1: cannot continue with installation!

Reply:
Done!